Packages/vcs/Lib/vtk_ui/text.py

```diff
@@ -87,45 +186,146 @@ def __init__(self, interactor, string, movable=False, on_move=None, on_drag=None
         else:
             self.actor = text_actor(string, fgcolor, size, font)
 
-        widget.SetTextActor(self.actor)
+        widget = WidgetReprShim(interactor, self.actor)
```

Comment:
Sounds good. I'm not terribly familiar with the widget classes, but the raw actor should provide any functionality you may need, with a lot more control.